### PR TITLE
sv.js - Community update, spell checked and corrected

### PR DESCRIPTION
**Pull request by Nubbie**
Hey
When I went to Gitter.im, I noticed how the Swedish translation was lacking strings. Another issue I've seen, some strings are translated in a 'weird' way and some even being grammatically incorrect.
Months ago, I asked about support for Android 4.4 and got a excellent answer which I wanted to redeem in a way, so this is my thanks for that excellent response :)
- Fully translated 
- Spell checked
- Corrections 
- Fixed grammatical errors

Have been translating strings over on a project called OpenRCT2, have translated the phone game BrainItOn and even done some translations over on Steam

### CODE REVIEW

File: homepage/sv.js [hunk 1]
```diff
@@ -2,66 +2,75 @@
...
 	"Developer friendly": "Utvecklarvänligt",
-	"This is a picture of a cloud against a background of clouds. OMG! We <a %s>integrate</a> with a bunch of stuff in the cloud.": "detta är en bild av ett moln, mot en bakgrund av moln. OMG! Vi <a %s>Integrerar</a> med en del grejer i molnet.",
+	"This is a picture of a cloud against a background of clouds. OMG! We <a %s>integrate</a> with a bunch of stuff in the cloud.": "Detta är en bild på ett moln, mot en bakgrund av moln. OMG! Vi <a %s>Integrerar</a> med massor i molnet.",
 	"Some of our communities": "Några av våra grupper",
-	"Create your own": "Skapa din egen",
+	"Create your own": "Skapa din egna",
 	"Explore rooms": "Utforska rum",
 	"Explore": "Utforska",
 	"Integrated with your workflow": "Integrerad med ditt arbetsflöde",
-	"Gitter is built on top of GitHub and is tightly integrated with your organisations, repositories, issues and activity.": "Gitter är byggt på GitHub och är nära integrerat med dina organisationer, repositories, issues och aktivitet.",
```
Comment (Nubbie) [inline homepage/sv.js]: 'Repositories' & 'Issues' is being skipped as it doesn't have a 'good official translation', most people know what it means anyways as it's GitHub related

File: homepage/sv.js [hunk 2]
```diff
@@ -2,66 +2,75 @@
...
+	"If you're ever around Silicon Roundabout, give us a shout, we love meeting up with other developers.": "Om du någonsin är runt Silicon Roundabout, kom förbi, vi älskar att träffa andra utvecklare.",
 	"Support": "Support",
 	"API": "API",
-	"Blog": "Blog",
+	"Blog": "Blogg",
 	"Twitter": "Twitter",
-	"Gitter &mdash; Where developers come to talk.": "Gitter &mdash; Where developers come to talk.",
-	"Translated By": "Översatt Av"
+	"Gitter &mdash; Where developers come to talk.": "Gitter &mdash; Vart utvecklare kommer för att snacka.",
+	"Translated By": "Mattias A (Nubbie on GitHub) & svbeon"
```
Comment (Nubbie) [inline homepage/sv.js]: Svbeon was the person starting the translation (GitHub name)

**Comment by Nubbie** [comment]
Noticed there was a community update! 
Will update the translation and include/change all the stuff that has been added/removed 
All the OLD strings will be commented out in the bottom _(For easy access if needed)_

**Comment by Nubbie** [comment]
The question is, something needed to push this forward?
As the status that's merged right now, could need a update

**Comment by Nubbie** [comment]
@MadLittleMods
Hey, this PR has been up for almost a year
Any way possible to get it merged?

**Comment by MadLittleMods** [comment]
Thanks for ping @Nubbie 😀 

This will be available when we update the dependency in the `webapp` and push out a release.

**Pull request merged**